Gemfile

```diff
@@ -12,7 +12,7 @@ gem "govspeak"
 gem "govuk_app_config"
 gem "govuk_document_types"
 gem "govuk_schemas"
-gem "govuk_sidekiq"
+gem "govuk_sidekiq", "6.0.0"
```

Comment:
Do we need to pin this to exactly version 6.0.0?

Reply:
Nope, it's against the rules: https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html

Reply:
woops, sorry